src/fips.c

```diff
@@ -28,6 +30,24 @@
 #include "ica_api.h"
 #include "test_vec.h"
 
+#ifndef PATH_MAX
```

Comment:
why ifndef ?

Reply:
Yes, in Libica it's currently the only occurrence of PATH_MAX. But as this constant is defined in linux limits.h, I thought better be safe ...

Reply:
okay